Validation/CTPPS/test/year_2016/proton_reconstruction_LHC_data/test_reconstruction_cfg.py

```diff
@@ -0,0 +1,79 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("CTPPSTestProtonReconstruction", eras.ctpps_2016)
+
+# declare global tag
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, "105X_dataRun2_relval_v2")
+#process.GlobalTag = GlobalTag(process.GlobalTag, "auto:run2_data")
```

Comment:
this earlier had an auto-GT configured.
why was a hardcoded value selected now?

We can't maintain compatibility with a hardcoded GT version in the future CMSSW versions.
So, this config looks dead on arrival.

would another auto-GT work?

Reply:
To my knowledge there is no auto GT which contains reasonable alignment, that's why the change.

Hopefully, our alignment will be propagated soon to the auto GTs, therefore I reverted back to `auto:run2_data` in 763708dc725.